transactions/update-contract/2022/aug-3/README.md

```diff
@@ -0,0 +1,67 @@
+# Update Service Account Contract
+
+> August 3, 2022
+
+## Contract to be deployed
+
+https://github.com/onflow/flow-core-contracts/blob/master/contracts/FlowToken.cdc
+
+## Dapper Labs Builds
```

Comment:
I copied from a previous upgrade here and just changed some names and dates. I'm not sure who should be the ones doing the building and signing or if we use a tool now